src/uu/base64/src/base64.rs

```diff
@@ -33,7 +37,8 @@ fn get_usage() -> String {
     format!("{0} [OPTION]... [FILE]", executable!())
 }
 
-pub fn uumain(args: impl uucore::Args) -> i32 {
+#[uucore_procs::gen_uumain]
+pub fn uumain(args: impl uucore::Args) -> UResult<()> {
```

Comment:
I think `base64` should be done together with `base32` and `base_common`, because there is more error handling hidden in `base_common`.

Reply:
Should I remove base64 from this PR?